sphinx/source/members/accept_recovery.rst

```diff
@@ -0,0 +1,112 @@
+Accepting Recovery and Submitting Shares
+========================================
+
+.. note:: Before members can initiate the end of the recovery procedure, operators should have started a new network and recovered all public transactions. See :ref:`details for public recovery operator procedure <operators/recovery:Establishing a Recovered Public Network>`.
+
+Accepting Recovery
+------------------
+
+Once the public recovered network has been established by operators, members are allowed to vote to confirm that the configuration of the new network is suitable to complete the recovery procedure.
+
+The first member proposes to recover the network:
```

Comment:
Maybe "a member proposes to recover the network" instead. `The first member` implies a specific member has to propose the network recovery. 

Also add: "and other members can vote on the proposal" before the voting example?

Reply:
Good point! Fixed now.